Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+# What I don't know, but should
+
+## Congratulations! You've joined Babylon...
+Welcome to the amazing iOS team! 
+Babylon is a fast-growing company. Currently, we are on-boarding ~30 new-starters weekly. Many of them are engineers like yourself. Our team is scattered across three offices: London West, London East and US office. On top of that, we have fully remote teammates. As you can imagine this poses almost biblical challenges, but don’t worry :)
```

Comment:
```suggestion
Babylon is a fast-growing company. Currently, we are on-boarding ~30 new-starters weekly. Many of them are engineers like yourself. Our iOS team is scattered across three offices: London West, London East, and the US office. On top of that, we have fully remote teammates. As you can imagine this poses almost biblical challenges, but don’t worry :)
```

We have more offices that have non-iOS people, so I added `iOS` to differentiate.